test/qweb3.js

```diff
@@ -512,4 +512,19 @@ describe('Qweb3', () => {
       });
     });
   });
+
+  describe('backupWallet()', () => {
+    it('backup the wallet', async () => {
+      const res = await qweb3.backupWallet('backup.dat');
+      assert.isDefined(res);
```

Comment:
does a successful backup return some sort of string vs if the backup failed? meaning, can we test a successful backup based on the returned result?